crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -363,6 +364,10 @@ pub fn derive_system_param(input: TokenStream) -> TokenStream {
     let manifest = Manifest::new().unwrap();
     let path_str = if let Some(package) = manifest.find(|name| name == "bevy") {
         format!("{}::ecs", package.name)
+    } else if let Some(package) = manifest.find(|name| name == "bevy_internal") {
+        format!("{}::ecs", package.name)
+    } else if let Some(package) = manifest.find(|name| name == "bevy_ecs") {
+        package.name
```

Comment:
this is already fixed with #1426

Reply:
Ah, I hadn't seen that one. I'll wait for it to get merged and then update my PR.